project/Prototypes.scala

```diff
@@ -36,7 +36,8 @@ trait Prototypes {
     cleanAll := Def.taskDyn {
       val allProjects = ScopeFilter(inAnyProject)
       clean.all(allProjects)
-    }.value
+    }.value,
+    unmanagedClasspath in Runtime += file("ui/dist")
```

Comment:
Unless this will only be temporary to get us started in which case that is fine... But lets add a note to remove.